pkg/agent/flowexporter/connections/connections.go

```diff
@@ -145,6 +155,39 @@ func (cs *ConnectionStore) addOrUpdateConn(conn *flowexporter.Connection) {
 				}
 			}
 		}
+
+		// Add network policy name and namespace from Connection label
+		if len(conn.Labels) != 0 {
+			// Make sure this is LittleEndian or BigEndian?
+			klog.V(4).Infof("connection label: %x; label masks: %x", conn.Labels, conn.LabelsMask)
+			ingressOfID := binary.LittleEndian.Uint32(conn.Labels[:4])
+			egressOfID := binary.LittleEndian.Uint32(conn.Labels[4:8])
+			// TODO: There's a chance that the ingressOfID is released and reused by another
+			//  NetworkPolicy rule could be modified in-between by the time the conntrack
```

Comment:
Guess you need to rephrase the sentence.

Reply:
Thanks for pointing this. This is not applicable after PR #1411 is merged, so removed it. I will be changing `asyncDeleteInterval` to `pollInterval` of Flow Exporter. 
Rephrased the below comment.